release-calendar/src/types.ts

```diff
@@ -26,6 +26,16 @@ export enum Channel {
   CREATED = 'created',
 }
 
+export const channels: Channel[] = [
```

Comment:
Having this and the `Channel` enum is confusing. Does this belong here or can it go to the `ChannelTable` component?